src/test/java/org/apache/mesos/testutils/NetworkTestUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.mesos.testutils;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+/**
+ * This class provides utilities useful for all tests which may have Network related needs.
+ */
+public class NetworkTestUtils {
```

Comment:
Why more than 2 places and not 1?  How would you solve the multiple uses of this functionality without factoring it out?